README.adoc

```diff
@@ -53,13 +53,18 @@ https://github.com/gosecure/malboxes
 === Debian
 
     apt install vagrant git python3-pip
+    
+=== Ubuntu
+
+    apt install git python3-pip
 
 == Installation
 
 === Linux/Unix
 
-* Install git, vagrant and packer using your distribution's packaging tool
+* Install git and packer using your distribution's packaging tool
   (packer is sometimes called packer-io)
+* Install vagrant from their website : https://www.vagrantup.com/downloads.html (Installing from some distributions' packaging tools have caused issues). 
```

Comment:
As this doesn't seem to be an issue on ArchLinux, I think this is Ubuntu specific and therefore would make more sense to add in the Ubuntu section. Also, would it be worth mentioning the Ubuntu version you are using since I think the behavior might change from one to another, right?

Thx for the help!

Reply:
I added this based on vagrant's documentation. They specifically mention not to install from package managers :
https://www.vagrantup.com/docs/installation/